recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Default set of sounds for AsteroidOS under CC0"
+HOMEPAGE = "https://github.com/AsteroidOS/asteroid-sounds"
+LICENSE = "CC0"
```

Comment:
`CC0` isn't a correct `SPDX` license (https://spdx.org/licenses/). Change it to `CC0-1.0`:
```
LICENSE = "CC0-1.0"
```